package.json

```diff
@@ -74,6 +75,9 @@
 		"webpack": "^5.76.0",
 		"yaserver": "^0.4.0"
 	},
+	"overrides": {
```

Comment:
`@typescript/ata` currently requires typescript 4.x as peerDependency.
If this PR is going to accepted and merged, I will make a PR to `@typescript/ata` to relax requirements.